### PR TITLE
Add enzyme

### PR DESCRIPTION
**Pull request by Rumaysa**
What?
- Added Enzyme and unit tests

Why?
- To be able to drive our code using tests(TDD) and ensure all necessary dependancies are installed

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -21,5 +21,8 @@
   },
   "include": [
     "src"
+  ],
+  "typeRoots": [
+    "./node_modules/@types", "./node_modules/jest-enzyme"
```
Comment (danielwilliams94) [inline tsconfig.json]: Do we know what this line does exactly? Just want to make sure we haven't added anything we don't need to 😄

Comment (PinkhamSoftware) [reply tsconfig.json]: Isn't that the typescript types for the jest module?

**Comment by PinkhamSoftware** [comment]
``` PASS  src/App.test.tsx
  ✓ renders without crashing (21ms)
  When using enzyme
    ✓ renders without crashing (4ms)

Test Suites: 1 passed, 1 total
Tests:       2 passed, 2 total
Snapshots:   0 total
Time:        5.309s
Ran all test suites.

Watch Usage: Press w to show more.
```

File: tsconfig.json (hunk 1 above)
Comment (PinkhamSoftware) [reply tsconfig.json]: @danielwilliams94 well spotted actually, [type root](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html) specifies that only those types are included and not anything else.

**Pull request merged**